weave/trace_server/interface/base_object_classes/annotation_column.py

```diff
@@ -0,0 +1,58 @@
+from typing import Optional
+
+import jsonschema
```

Comment:
did you need to add this package? I wonder if the trace server already has this transitively.

Reply:
I dont understand what this means... I see that the trace server has it in the requirements, how do I access it "transitively"?

Reply:
yeah, this is what I mean. in our pyproject.toml (inside core), there is no `jsonschema`. However, since `litellm` depends on `jsonschema`, you get it for free in trace server (when deployed).

Reply:
but for these CI tests to pass you need to add jsonschema to the dev reqs

Reply:
inside of 
```
test = [
  "nox",
  "pytest>=8.2.0",
  "pytest-asyncio>=0.23.6",
  "pytest-cov>=5.0.0",
  "pytest-xdist>=3.1.0",
  "pytest-rerunfailures>=12.0",
  "pytest-rerunfailures>=14.0",
  "clickhouse_connect==0.7.0",
  "fastapi>=0.110.0",
  "sqlparse==0.5.0",

  # Integration Tests
  "pytest-recording>=0.13.2",
  "vcrpy>=6.0.1",

  # serving tests
  "flask",
  "uvicorn>=0.27.0",
  "pillow",
  "filelock",
  "httpx",
]
```

Reply:
Yeah got it thanks